static/sass/_v1_pattern_card.scss

```diff
@@ -0,0 +1,15 @@
+@mixin ubuntu-p-card {
+  .p-card__footer--upgrade {
+    background: $color-light url('#{$assets-path}f68488b1-picto-upgrade-warmgrey.svg') 20px center no-repeat;
+    background-size: 40px 40px;
```

Comment:
We should avoid using pixel values in CSS. Ideally, you should use [Vanilla spacing variables](https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_settings_spacing.scss) - in this case `padding: $sp-xx-large $sp-xx-large`.

Where there is not an exact 1:1 match, choose the nearest value. If that value breaks the design, you may then be justified in using an alternate `rem` value but this is the last resort option.